highfidelity/hifi
# Android - Prevent pending downloads to happen when going back to home

Downloads shouldn't start when going back to home (Android)

**Test Plan**

1. Open the app
2. Enter a Domain
3. Before it loads all, expect so see one object loaded and press back immediately
4. Domains list should appear
5. Enter again to the same Domain

**Verify**

- It was experienced in previous versions that in some cases, domains list loading was slow after coming back from Interface (~15s). It shouldn't be that slow now (consider a 3s as max with decent bandwidth).
- Be sure we don't get stuck without downloading (download broken). If we try to enter again the domain after coming back, elements should resume downloading.
